src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupOperation.cs

```diff
@@ -139,11 +140,12 @@ public void SetBackupInput(BackupInfo input)
         /// <returns></returns>
         public BackupConfigInfo GetBackupConfigInfo(string databaseName)
```

Comment:
>GetBackupConfigInfo [](start = 32, length = 19)

nit: typically methods that create new objects are named CreateXXXX and methods that retrieve existing things are GetXXXX.  Methods that either create if not existing or get existing are GetOrCreateXXXX.  I think all or most the GetXXXX methods in this class would be better named as Create methods